src/styles/index.scss

```diff
@@ -1,3 +1,164 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: block;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  box-sizing: border-box;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+  padding: 0;
+}
+
+.card__content {
+  margin: 0 16px 16px;
+  padding: 0;
+  width: 166px;
+}
+
+.card__title {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.card__description {
+  font-style: normal;
+  margin: 0;
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.stars__item {
```

Comment:
Following BEM, each block should have own `css` file. So I suggest create `card.scss` and `stars.scss` files